plugin/keyring_vault/vault_keyring.cc

```diff
@@ -13,32 +13,13 @@ using keyring::Vault_keys_container;
 using keyring::Vault_curl;
 using keyring::Logger;
 
-CURL *curl = NULL;
 mysql_rwlock_t LOCK_keyring;
 
-static bool init_curl()
-{
-  curl_global_init(CURL_GLOBAL_ALL);
-  curl = curl_easy_init();
-  if (curl == NULL)
-  {
-    logger->log(MY_ERROR_LEVEL, "Could not create CURL session");
-    return true;
-  }
-  return false;
-}
-
-static void cleanup_curl()
-{
-  if (curl != NULL)
-    curl_easy_cleanup(curl);
-  curl_global_cleanup();
-}
 
 static bool reset_curl()
```

Comment:
Do we still need to call `reset_curl()` from `check_keyring_file_data()` (when changing `keyring_vault_config`)?

Reply:
Yes, this is needed for Centos6. For some reason without this CURL (from 2008) just crashes.